File: mot-selenium/src/main/java/com/dvsa/mot/selenium/datasource/enums/Role.java
package com.dvsa.mot.selenium.datasource.enums;

public enum Role {
    AED("AED", "Authorised examiner delegate", "Authorised examiner delegate", "AED","2"),
    AEDM("AEDM", "AEDM", "Authorised Examiner Designated Manager", "AEDM","1"),
    SITE_MANAGER("Site Manager", "Site Manager", "Site Manager", "SITE-MANAGER",""),
    SITE_ADMIN("Site Admin", "Site Admin", "Site Admin", "SITE-ADMIN",""),
    TESTER("Tester", "Tester", "Tester", "TESTER","");

    private final String name;
    private final String shortName;
    private final String fullName;
    private final String assignRoleName;
    private final String roleId;

    public String getName() {
        return name;
    }

    public String getShortName() {
        return shortName;
    }

    public String getFullName() {
        return fullName;
    }

    public String getAssignRoleName() {
        return assignRoleName;
    }

    public String getRoleId() {
        return roleId;
    }

    private Role(String name, String shortName, String fullName, String assignRoleName , String roleId) {
        this.name = name;
        this.shortName = shortName;
        this.fullName = fullName;
        this.assignRoleName = assignRoleName;
        this.roleId = roleId;
    }
}

File: mot-web-frontend/module/DvsaClient/src/DvsaClient/Entity/Role.php
<?php

namespace DvsaClient\Entity;

/**
 * Class Role.
 */
class Role
{
    protected $id;
    protected $name;
    protected $fullName;
    protected $shortName;

    /**
     * @param string $fullName
     *
     * @return $this
     */
    public function setFullName($fullName)
    {
        $this->fullName = $fullName;

        return $this;
    }

    /**
     * @return string
     */
    public function getFullName()
    {
        return $this->fullName;
    }

    /**
     * @param int $id
     *
     * @return $this
     */
    public function setId($id)
    {
        $this->id = $id;

        return $this;
    }

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param string $name
     *
     * @return $this
     */
    public function setName($name)
    {
        $this->name = $name;

        return $this;
    }

    /**
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * @param string $shortName
     *
     * @return $this
     */
    public function setShortName($shortName)
    {
        $this->shortName = $shortName;

        return $this;
    }

    /**
     * @return string
     */
    public function getShortName()
    {
        return $this->shortName;
    }

    /**
     * @return string
     */
    public function getNameForDisplay()
    {
        return $this->getFullName().' ('.$this->getShortName().')';
    }
}
